docs/installing.md

```diff
@@ -1,104 +1,58 @@
 # Running a local Haveno test network
 
-These are the steps needed to set up a local Haveno test network.
+ These are the steps needed to set up a local Haveno test network.  You'll have the possibility to either connect to our shared Monero stagenet node or to create your own local environment.
 
-## 1. Download dependencies
+## 1. Install dependencies
 
-1. Install git, git-lfs, and Java. On Ubuntu: `sudo apt install git git-lfs openjdk-11-jdk`
-2. Download the Monero binaries. To make things easier, we provide two custom Monero binaries: `monerod` and `monero-wallet-rpc`. You can directly [download the software (Linux and Mac)](https://github.com/haveno-dex/monero/releases/tag/testing) or build the binaries yourself by cloning and building the `release-v0.17` branch of this repository: https://github.com/haveno-dex/monero
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download).
+On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Bitcoin and Monero binaries will be downloaded and verified automatically in the next step.
```

Comment:
No need to mention the binaries that will be set up in the next step if it's not relevant or there's no action to be taken in this step.